### ISSUE TITLE
Feature request: add multiple clusters using environment variables

### ISSUE THREAD

**Issue by jrglee** [ElasticHQ/elasticsearch-HQ#406]
The env var `HQ_DEFAULT_URL` is very useful, but often orgs have to manage multiple Elasticsearch clusters. It would be good if the list of clusters to connect too could be setup upfront.

The other reason why this would be helpful is for users who deploy ElasticHQ in environments like Kubernetes, where the sqlite db file is ephemeral. I am not sure if you have MySQL/Postgress or other relational databases on the roadmap. If yes, then this feature does not make sense.

**Comment by royrusso** [comment]
Good idea. This is a common issue/downside to the way I'm using sqllite. When restarting the container, the DB gets wiped. I'm not sure how to address this in a clean way. Any ideas from the community would be helpful. 

I've considered external DB support. So on startup, HQ would know to connect to an existing RDBMS somewhere on the network to get it's cluster information.
